### ISSUE TITLE
Nowy endpoint API wysyłkowego - numer ulicy

### ISSUE THREAD

**Issue by it-cewe-pl** [allegro/allegro-api#9365]
Witam,
czy numer ulicy jest rzeczywiście wymagany ? 
Ewentualnie co można tam wpisać zamiast rzeczywistego numeru ulicy ?

Ponieważ enpoint _https://api.{environment}/order/checkout-forms/{id}_
zwraca w sekcji delivery.address, tylko street, bez street number, a nie sposób ogarnąć wszystkich sposóbów wpisywania adresów przez użytkowników. Tak aby w prosty i uniwersalny sposób wyodrębić numer ulicy z jej nazwy. 

Chyba, że niedalekiej przyszłości planowane jest zwarcanie tej wartość poprzez API ?

Pozdrawiam
Dawid Cz.

**Comment by Lukasz-Zurek** [comment]
Tak, numer ulicy jest wymagany. Dopytuję tutaj jednak odpowiedniego zespołu w kwestii braku spójności z formatem adresu w zamówieniach. Jak tylko uzyskam dodatkowe informacje, podzielę się nimi w tym wątku.

**Comment by PrzemyslawSegrecki** [comment]
Podłączam się pod pytanie.
Jak mamy uzupełniać street i streerNumber?

**Comment by Lukasz-Zurek** [comment]
@PrzemyslawSegrecki @it-cewe-pl Na chwilę obecną faktycznie na liście zamówień (jak i w szczegółach zamówienia) nie rozbijamy adresu na street i streetNumber. Rozbijamy je na etapie tworzenia przesyłek z uwagi na to, że niektórzy przewoźnicy przyjmują osobno ulicę i numer. Nie przewidujemy zmian w tym zakresie.

Zachęcam Was jednak do oddania głosów na istniejący już wątek z sugestią zmian w zamówieniach, pod [tym adresem](https://github.com/allegro/allegro-api/discussions/9218).
